sources/mongodb/helpers.py

```diff
@@ -12,10 +13,16 @@
 from dlt.common.typing import TDataItem
 from dlt.common.utils import map_nested_in_place
 from pendulum import _datetime
+import pyarrow
 from pymongo import ASCENDING, DESCENDING, MongoClient
 from pymongo.collection import Collection
 from pymongo.cursor import Cursor
 
+import pymongoarrow
```

Comment:
pyarrow and pymongoarrow should be imported in ` CollectionArrowLoader` only when they are used.
please import arrow from `dlt.common.libs.pyarrow`. this handles missing dependencies nicely

Reply:
Done